lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -1934,14 +1953,13 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
             }
         });
 
-        it("throws error if cacheLocation is Memory Storage and storeAuthStateInCookie is false", async () => {
```

Comment:
This test ensures that redirects are blocked if using memory storage since we do not support it